### DISCUSSION TITLE
Add Order to Col

### DISCUSSION THREAD

**Discussion by Mahdibayat** [rsuite/rsuite#2474] [Q&A]
i want to add Order to Col Tag

But I can't, don't see any properti

**Comment by SevenOutman** [comment]
Hi @Mahdibayat, could you explain in detail what you meant by "add order to col tag"?

**Reply by Mahdibayat** [reply]
🙄

**Comment by Mahdibayat** [comment]
> Hi @Mahdibayat, could you explain in detail what you meant by "add order to col tag"?

Hi @SevenOutman 
in bootstrap when we want in "lg" size col one stay in top (upper than other), but in "xs" size want col stay in bottom of other col, we use "order" property.
Can we do this with rsuite 🧐
Thanks 🙏

**Reply by SevenOutman** [reply]
The [push and pull utilities](https://rsuitejs.com/components/grid/#push-and-pull) may meet your needs.

**Reply by Mahdibayat** [reply]
I'm try Pull and push but, don't go next Line (Row)
